spreadsheet_oca/views/spreadsheet_spreadsheet.xml

```diff
@@ -78,12 +110,101 @@
         </field>
```

Comment:
Noticed you added inherited "mail.thread", "mail.activity.mixin" to spreadsheet_spreadsheet.py but didn't enable chatter in the view. Think this would be a welcome simple change. Just add this line after the ending sheet tag:

`<chatter/>`